services/apps/src/apps/routes/schemas/groups.clj

```diff
@@ -0,0 +1,72 @@
+(ns apps.routes.schemas.groups
+  (:use [common-swagger-api.schema :only [describe NonBlankString]])
+  (:require [schema.core :as s]))
+
+(s/defschema Group
```

Comment:
Some of the schemas in this namespace seem like duplicates of [schemas in iplant-groups](https://github.com/cyverse/DE/tree/8fba9b30622796c94e8003a051e77422aef1b931/services/iplant-groups/src/iplant_groups/routes/schemas).
These common schemas could be moved into the `common-swagger-api` lib so that iplant-groups and apps can share them.
For example, [some ontology schemas](https://github.com/cyverse/DE/tree/8fba9b30622796c94e8003a051e77422aef1b931/libs/common-swagger-api/src/common_swagger_api/schema/ontologies.clj) are shared between the apps and metadata services.